### PR TITLE
Experimental tutorial on integers

### PR DESCRIPTION
**Pull request by konnov**
This is an experimental tutorial that introduces operators on integers. The integers are not hard to understand. The main purpose is to try two approaches to writing tutorials:

 - a CodeTour tutorial: `tutorials/.tours/integers.tour`
 - a markdown tutorial: `tutorials/lesson2-integers/integers.md`

The both tutorials are generated from `tutorials/lesson2-integers/integers.template.qnt`. Please add comments to this template, except commenting on the generated files.

The files are generated by an ad hoc script `tutorials/script/mk-tutorial.py`. I would ask you not to focus too much on this script. Hopefully, it will be replaced by a systematic solution later. I quickly hacked it in python, to try the idea, not to design a good piece of code.

Finally, the tutorial mentions secret codes. They are not implemented in the tutorial yet. I will add them, once this PR is merged.

- [x] Entries added to the respective `CHANGELOG.md` for any new functionality
- [x] Feature table on [`README.md`](../README.md#roadmap) updated for any listed functionality

### CODE REVIEW

File: tutorials/lesson2-integers/integers.template.qnt [hunk 1]
```diff
@@ -0,0 +1,337 @@
...
+    !*/
+
+    // 0 is an integer literal
+    pure val int0 = 0
+
+    // 1 is an integer literal
+    pure val int1 = 2
+
+    // 2 is an integer literal
+    pure val negative1 = -3
```
Comment (bugarela) [inline tutorials/lesson2-integers/integers.template.qnt:40]:
```suggestion
    // 0 is an integer literal
    pure val int0 = 0

    // 2 is an integer literal
    pure val int1 = 2

    // -3 is an integer literal
    pure val negative1 = -3
```

File: tutorials/lesson2-integers/integers.template.qnt [hunk 2]
```diff
@@ -0,0 +1,337 @@
...
+
+We omit several integer operators that produce sets. These operators are covered in
+the tutorial on sets.
+        </description>
+      </step>
+     !*/
+
+    // i^j is the integer exponentiation, that is,
+    // `i` multiplied by itself `j - 1` times
+    pure def myPow(i, j) = i^j
```
Comment (bugarela) [inline tutorials/lesson2-integers/integers.template.qnt:64]: In the markdown version, this is shown after the examples, while it seems like it should appear before them.

File: vscode/quint-vscode/server/package.json [hunk 3]
```diff
@@ -12,7 +12,7 @@
 		"url": "https://github.com/informalsystems/quint"
 	},
 	"dependencies": {
-        "@informalsystems/quint": "^0.4.0",
+		"@informalsystems/quint": "file:.yalc/@informalsystems/quint",
```
Comment (bugarela) [inline vscode/quint-vscode/server/package.json:15]: This is a local change made by `make local` that is not supposed to be commited.

```suggestion
		"@informalsystems/quint": "^0.4.0",
```

**Review by bugarela — APPROVED** [review]
I've already voiced my opinion on tutorials for basic concepts, so I don't think I have much to add in this review. Like you said: let's try it out and see what people think :smile:

File: tutorials/lesson2-integers/integers.template.qnt (hunk 2 above)
Comment (konnov) [reply tutorials/lesson2-integers/integers.template.qnt:64]: Yes, I have found this way more natural. In Codetour, the explanation looks better below the code, whereas in Markdown the code looks better as an example.

**Pull request merged**